pipelines/noaa_cns_wind/config/dataset.yaml

```diff
@@ -0,0 +1,141 @@
+attrs:
+  title: noaa_cns_wind
+  description:
+    Taking data for Cns Wind data from CA and converting it from ascii text to
```

Comment:
Please go ahead and describe the data here, not necessarily the pipeline. You can probably get this off of their website